### ISSUE TITLE
rethink slogan

### ISSUE THREAD

**Issue by webratte** [kontalk/website#13]
Kontalk is not really "a NEW way of communicating"

What Do you think about:
"a safe way of communicating"?

**Comment by daniele-athome** [comment]
Well it can be thought of "new" because of the new way of using XMPP with phone numbers and improved federation. It's not only the safety ;-)

**Comment by webratte** [comment]
Of course your right. But "normal" users are not interested in XMPP or what ever.
They interested in UI, chatting and hopefully in safety and privacy.
That's the big difference between Kontalk and  WhatsApp.
Isn't it ;-)

**Comment by daniele-athome** [comment]
That is also right. I'm going to rethink some of the presentation concepts in my talk and get them a go to my audience, let's see what comes out.

**Comment by marcelklehr** [comment]
I agree with @webratte I'd choose something along the lines of "Instant messaging by the people for the people" -- just my two cents.

**Comment by daniele-athome** [comment]
And I totally forgot about this :-) thanks for your suggestions guys, kontalk.org template renewal will come along with some brand changes.

**Comment by webratte** [comment]
@marcelklehr This sounds good for me. 
It brings the community in focus. 👍

**Comment by webratte** [comment]
@marcelklehr This sounds good for me. 
It brings the community in focus. :+1:

**Comment by tetris4** [comment]
Just adding some further suggestions to this brainstorming as I had also given this some thought but just now noticed this thread:

- A community driven messenger application.
- A communication tool by the community for the community.
- Collectively developed communication (together?) 

I think it is important for the slogan to express what distinguishes Kontalk from other messenger applications.
